docs/docs/running-hello-world.md

```diff
@@ -73,216 +75,235 @@ The web server loads the enclave and waits for requests from the client. Amongst
+Try rerunning the command with a different signing key:
 ```
-"S:2222222222222222222222222222222222222222222222222222222222222222 PROD:1 SEC:INSECURE"
+java -jar client/build/libs/client.jar "S:2222222222222222222222222222222222222222222222222222222222222222 PROD:1
+SEC:INSECURE" reverse-me
 ```
 
-You should see the following error:
+You will get the following error, as the client's signing key(code signer) does not match the enclave's signing key:
 > com.r3.conclave.common.InvalidEnclaveException: Enclave code signer does not match any of the acceptable code signers.
 (key hash 0000000000000000000000000000000000000000000000000000000000000000 vs acceptable
 2222222222222222222222222222222222222222222222222222222222222222)
 
-For more information on how to choose a constraint, see [Enclave constraints](constraints.md).
-
+For more information on choosing a constraint, see [Enclave constraints](constraints.md).
 
 ## Beyond mock mode
-Eventually you will want to run your application in one of the other [enclave modes](enclave-modes.md), namely simulation,
-debug, or release. This section describes how to build and run the sample in these other modes.
+This section describes how to build and run the sample in other [enclave modes](enclave-modes.md), namely
+simulation, debug, and release.
 
-**First, make sure that your system fulfills the
-[system requirements](enclave-modes.md#system-requirements) for your desired mode.**
+*First, ensure that your system fulfills the [system requirements](enclave-modes.md#system-requirements) for
+your desired mode.*
 
 ### Build the enclave in other modes
 
-The hello world sample has been [configured](enclave-modes.md#set-the-enclave-mode) such that you can supply the mode
-via the `enclaveMode` Gradle parameter. For example, you could compile the host for simulation mode with the command
+The hello world sample has been [configured](enclave-modes.md#set-the-enclave-mode) such that you can define the mode
+using the `enclaveMode` Gradle parameter. For example, you can compile the host for simulation mode with the command
+
 === "Windows"
-    ```bash
+```bash
     gradlew.bat :host:bootJar -PenclaveMode=simulation
-    ```
+```
 === "macOS / Linux"
-    ```bash
+```bash
     ./gradlew :host:bootJar -PenclaveMode=simulation
-    ```
+```
+
+This generates the host JAR at `host/build/libs/host-simulation.jar`.
 
-This would generate the host JAR at `host/build/libs/host-simulation.jar`.
+!!!Note
 
-!!! tip
-    If working from inside IntelliJ, find the bootJar task of your host project from the Gradle menu on the right-hand 
-    side. Modify run configuration of this task by adding the -PenclaveMode=<MODE> flag to the arguments section.
+    If you use IntelliJ, find the bootJar task of your host project from the Gradle menu on the right-hand 
+    side. Modify the run configuration of this task by adding the -PenclaveMode=<MODE> flag to the arguments section.
     Now you can run the task.
```

Comment:
I don't think we should be providing such guidance on Intellij. But I leave it up to you.

Reply:
Same thinking here. We used to have screenshots of how to build hello world in IntelliJ and in that case this note made sense. But as we now show all the commands in the command line, this doesn't really make sense/doesn't suit here.

Reply:
Removed.